loadtest/usecases/2_goto-sp-saml.ts

```diff
@@ -0,0 +1,18 @@
+import { fail, sleep } from "k6";
+import { getDefaultOptions } from "../util/config.ts";
+import { loadPage, navigateToHomepage } from "../util/page.ts";
+import { getDefaultAdminMix } from "../util/users.ts";
+
+export const options = {
+  ...getDefaultOptions(),
+};
+
+const serviceProviderName = "School-SH";
+
+export default function main(users = getDefaultAdminMix()) {
```

Comment:
Should this use the `defaultAdminMix`? 
Sending >250 requests to School-SH is a lot

Reply:
The naming/configuration is a bit confusing here: The UserMix actually gives a ratio of user-categories, not the number of actual requests or users. The number of virtual users is defined via the exported options. See `loadtest\util\config.ts`.

I am not sure about the actual numbers yet. Current configuration would have up to 100 virtual users going through this use-case at the same time, but I actually expected the number to be much higher than this. Do you have a different estimate for the load?

Reply:
Ah, I didn't know those weren't actual numbers. No idea what realistic estimates would be, but it just feels like sending all those users to an external service (in this case School-SH) would be load-testing *their* infrastructure instead of ours.

Reply:
Yes, that is a concern I have raised multiple times as well. The reason is that School-SH sits behind a gateway and it was explicitly requested that this ALG had to be hit as well, because there are concerns about its reliability.

Reply:
If that concern is known and accepted then this is fine by me 👍